src/app/components/Navbar.jsx

```diff
@@ -31,10 +36,21 @@ const NavItem = styled.li`
 `
 
 const SignOutButton = styled.button`
-  background: none;
+  background: linear-gradient(0deg, rgba(217, 217, 217, 0.1), rgba(217, 217, 217, 0.1)),
+    linear-gradient(97.1deg, rgba(247, 225, 255, 0.38) 11.37%, rgba(218, 115, 255, 0.38) 102.95%);
+
   border: none;
   color: white;
   cursor: pointer;
+  border: 1.85px solid;
+  border-radius: 35px;
+  padding: 12px 60px;
+
+  border-image-source: linear-gradient(
+    97.1deg,
+    rgba(247, 225, 255, 0.38) 11.37%,
+    rgba(218, 115, 255, 0.38) 102.95%
+  );
```

Comment:
Keep in shared folder

Reply:
commit link [commit](https://github.com/ashutosh-rath02/project-burrrah/pull/8/commits/bd1c572c68e5a7b3e5ce39b22aa65d0fe1dfc104)